packages/appstore/src/components/transaction-list/transaction-list.tsx

```diff
@@ -1,74 +1,34 @@
-import React from 'react';
-import { useFetch } from '@deriv/api';
+import React, { useState } from 'react';
 import { Statement } from '@deriv/api-types';
 import { getCurrencyDisplayCode, isCryptocurrency } from '@deriv/shared';
-import { Text } from '@deriv/components';
+import { Text, Dropdown } from '@deriv/components';
 import { useGroupedFiatTransactions } from '@deriv/hooks';
 import { useStore } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import { getWalletCurrencyIcon } from 'Constants/utils';
 import FiatTransactionListItem from './fiat-transaction-list-item';
-import './fiat-transaction-list.scss';
+import './transaction-list.scss';
 
-const FiatTransactionList = () => {
+const useFetch = (name: string, props: any) => {
```

Comment:
@rostislav-deriv Why not create a `useWalletStatement` hook or something like that and abstract things there? 🤔 You can actually do the API call and then change the response in the way you want, Check `useWalletList` hook as an example 🙇🏻

Reply:
Done that 🙇